src/flamenco/runtime/fd_rocksdb.c

```diff
@@ -433,12 +433,13 @@ fd_rocksdb_copy_over_txn_status_range( fd_rocksdb_t *    src,
                                        ulong             start_slot,
                                        ulong             end_slot ) {
   /* Look up the blocks data and iterate through its transactions */
-  fd_block_map_t * block_map = fd_blockstore_block_map( blockstore );
-  fd_wksp_t * wksp = fd_blockstore_wksp( blockstore );
+  fd_block_map_map_t * block_map  = fd_blockstore_block_map( blockstore );
+  fd_block_map_t *     block_pool = fd_blockstore_block_map_pool( blockstore );
+  fd_wksp_t *          wksp       = fd_blockstore_wksp( blockstore );
 
   for ( ulong slot = start_slot; slot <= end_slot; ++slot ) {
     FD_LOG_NOTICE(("fd_rocksdb_copy_over_txn_status_range: %d", slot));
-    fd_block_map_t * block_entry = fd_block_map_query( block_map, &slot, NULL );
+    fd_block_map_t * block_entry = fd_block_map_map_ele_query( block_map, &slot, NULL, block_pool );
```

Comment:
this probably shouldn't be using the internal implementation in the first place, should be calling `fd_blockstore_block_map_query` API